ext/freebsd-getopt/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# CMakeLists.txt - Getopt Library for Zephyr
+
+zephyr_include_directories(.)
+zephyr_library()
+zephyr_library_include_directories(${ZEPHYR_BASE}/include/posix)
```

Comment:
This should not be needed.
Sounds like this library has a dependency to `POSIX_API`, has that been ensured in Kconfig ?

If `CONFIG_POSIX_API=y` you can do a:
```
zephyr_library_link_libraries(posix_subsys)
```

Reply:
I've added `depends on POSIX_API` to `Kconfig`.
And I've changed
`zephyr_library_include_directories(${ZEPHYR_BASE}/include/posix)`
to
`zephyr_library_link_libraries(posix_subsys)`
in all iperf3, curl and getopt.